redhat-developer/app-services-cli
# refactor: rename packages

### Verification Steps

1. Run `make binary`
2. Run `rhoas version` - it should work.

### Type of change


### Checklist
